src/Http/Cookie/Cookie.php

```diff
@@ -118,6 +126,7 @@ class Cookie implements CookieInterface
      * @param string $domain Domain
      * @param bool $secure Is secure
      * @param bool $httpOnly HTTP Only
+     * @param null $sameSite Samesite
```

Comment:
...|

Reply:
Sorry, I fail to see what you mean. Do you mean | string?

Reply:
Cool, thank you